### PR TITLE
[ Cleanup ]  cleaning up the code 

### PR DESCRIPTION
**Pull request by mattisonchao**
### Motivation

Cleanup some code

### Modifications

- Fixed incorrect use logger.
- Use Awaitility to instead of Thread sleep

### CODE REVIEW

File: pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java [hunk 1]
```diff
@@ -65,8 +65,8 @@ public static EventLoopGroup newEventLoopGroup(int nThreads, boolean enableBusyW
                     try {
                         CpuAffinity.acquireCore();
                     } catch (Throwable t) {
-                        log.warn("Failed to acquire CPU core for thread {}", Thread.currentThread().getName(),
-                                t.getMessage(), t);
+                        log.warn("Failed to acquire CPU core for thread {}, the error message is {}",
```
Comment (eolivelli) [inline pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java:68]: log.warn will print the stacktrace when you pass a Throwable argument, so no need for "the error message is {}"

File: pulsar-common/src/test/java/org/apache/pulsar/common/util/collections/GrowableArrayBlockingQueueTest.java [hunk 2]
```diff
@@ -131,11 +132,8 @@ public void blockingTake() throws Exception {
                 queue.put(n);
                 ++n;
             }
-
             // Wait until all the entries are consumed
-            while (!queue.isEmpty()) {
-                Thread.sleep(1);
-            }
+            Awaitility.await().until(queue::isEmpty);
```
Comment (eolivelli) [inline pulsar-common/src/test/java/org/apache/pulsar/common/util/collections/GrowableArrayBlockingQueueTest.java:136]: Awaitility will not pool every millisecond by default
this change will slow down the test and probably is is overwhelming
this is a test about a BlockingQueue, it is fine to have a Thread.sleep here
it is not a cause of flaky tests

**Review by eolivelli — CHANGES_REQUESTED** [review]
I am not sure we need this change, see my comments

thank you for improving Pulsar codebase

File: pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java (hunk 1 above)
Comment (mattisonchao) [reply pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java]: I already fix it. :)

File: pulsar-common/src/test/java/org/apache/pulsar/common/util/collections/GrowableArrayBlockingQueueTest.java (hunk 2 above)
Comment (mattisonchao) [reply pulsar-common/src/test/java/org/apache/pulsar/common/util/collections/GrowableArrayBlockingQueueTest.java]: I already fix it.

**Comment by mattisonchao** [comment]
@eolivelli PTAL 👍🏼

File: pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java [hunk 3]
```diff
@@ -65,8 +65,7 @@ public static EventLoopGroup newEventLoopGroup(int nThreads, boolean enableBusyW
                     try {
                         CpuAffinity.acquireCore();
                     } catch (Throwable t) {
-                        log.warn("Failed to acquire CPU core for thread {}", Thread.currentThread().getName(),
-                                t.getMessage(), t);
+                        log.warn("Failed to acquire CPU core for thread {}",Thread.currentThread().getName(), t);
```
Comment (eolivelli) [inline pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java:68]: Missing one space in ",Thread":
`log.warn("Failed to acquire CPU core for thread {}", Thread.currentThread().getName(), t);`

**Review by eolivelli — CHANGES_REQUESTED** [review]
please change the title and the description accordingly

thank you for you contribution and for cleaning up the code base

File: pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java (hunk 3 above)
Comment (mattisonchao) [reply pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java]: I will fix it.

**Comment by mattisonchao** [comment]
@eolivelli  PTAL

**Pull request closed without merging**